.github/workflows/ci.yaml

```diff
@@ -78,6 +78,15 @@ jobs:
       - name: run tests
         run: ./scripts/ci/wasm_tests
 
+  wasm_tests_storage_v2:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install wasm-pack
+        run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+      - name: run tests
+        run: ./scripts/ci/wasm_tests
```

Comment:
Should this be the v2 script?

Reply:
It should have been, I actually fixed this in the latest rebase but it's also somewhat irrelevant now that we're removing the storage-v2 feature flag.